### DISCUSSION TITLE
Installing understrap on multisite

### DISCUSSION THREAD

**Discussion by JamesVanWaza** [holger1411/understrap#1505] [Q&A] [answered]
Greetings, I installed understrap and understrap child theme in Wordpress multisite, but the understrap child theme keeps showing the following error. 

<img width="1189" alt="Screen Shot 2021-11-13 at 9 49 55 PM" src="https://user-images.githubusercontent.com/4000076/141665550-68fe7b9f-40c7-4015-9fa1-35fe19c99914.png">
<img width="1190" alt="Screen Shot 2021-11-13 at 9 50 11 PM" src="https://user-images.githubusercontent.com/4000076/141665552-e67063e0-0d94-4dd4-8156-9721d35410b5.png">

It works really well on single sites without issue.

**Comment by bacoords** [comment]
We run Understrap with a few separate child themes on a multisite here: http://demos.understrap.com

Can you show us the 'All' tab on that top screenshot? It appears that Understrap (parent theme) isn't available across the network on that instance?

**Comment by JamesVanWaza** [comment]
Hi @bacoords
<img width="906" alt="Screen Shot 2021-11-15 at 4 19 34 PM" src="https://user-images.githubusercontent.com/4000076/141855285-1cf66fbb-4e91-4cb3-94f6-f1aff69a4bf6.png">

**Comment by JamesVanWaza** [comment]
<img width="1326" alt="Screen Shot 2021-11-15 at 4 21 12 PM" src="https://user-images.githubusercontent.com/4000076/141855465-0a2fdc92-aea1-472a-98de-26577226880e.png">

**Comment by bacoords** [comment, marked as answer]
Ah - it looks like the parent theme is installed in a directory as `understrap-1.0.1`, but the theme will be looking for the directory name to be `understrap`.

**Reply by JamesVanWaza** [reply]
Yes, that helped when I changed the name. 

<img width="898" alt="Screen Shot 2021-11-15 at 4 35 26 PM" src="https://user-images.githubusercontent.com/4000076/141857115-ee9e30d1-3923-43e8-ac71-ef0d26311614.png">
